sbin/xivo-check-master-status

```diff
@@ -2,15 +2,28 @@
 ip_addr=$1
 ping_count="3"
 ping_interval="10"
+confd_port=${CONFD_PORT:-9486}
+confd_version=${CONFD_API_VERSION:-1.1}
 
 if [ -z "$ip_addr" ]; then
     echo "usage: $(basename $0) ip_addr"
     exit 1
 fi
 
+check_ha_mode(){
+    local token=$(wazo-auth-cli token create --expiration 20)
+    curl -k -H "X-Auth-Token: $token" http://localhost:$confd_port/$confd_version/ha | jq -r .node_type
+}
+
 ping -c $ping_count -i $ping_interval $ip_addr
-if [ $? -eq 0 ]; then
-    /usr/sbin/xivo-manage-slave-services stop
+ping_status=$?
+if [ $(check_ha_mode) != "slave" ]; then
+    echo "HA mode changed, this slave has been freed!" >&2
```

Comment:
Please don't. Jokes are unwelcome when there is an emergency. Plus the master/slave naming scheme is already a bad one and that kind of reference only makes it worse.

Reply:
Yes you right, so probably we can rename sentences with primary and secondary. On the other hand, I'm not sure we want to rename master/slave everywhere in the code.

Reply:
Wasn't really a joke, just a reference to the concepts of master and slave that accurately describe the situation(which is why those words are used in analogy in the first place).
But understood, I'll change the words in the text, though that might introduce confusion if we are using two nomenclatures simultaneously in the public APIs.